src/main/java/Pat.java

```diff
@@ -0,0 +1,172 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Pat {
+    // ANSI escape codes to change format chat
+    public static final String ANSI_RED = "\033[31m";
+    public static final String ANSI_PINK = "\033[95m";
+    public static final String ANSI_RESET = "\033[0m";
+    public static final String ANSI_ERASE_LINE = "\033[2K";
+    public static final String ANSI_MOVE_TO_START_OF_PREVIOUS_LINE = "\033[F";
+    public static final String INDENT = "    ";
+
+    private static final ArrayList<Task> todoList = new ArrayList<>();
+
+    private static void say(String message) {
+        System.out.println(ANSI_PINK + message + ANSI_RESET);
+    }
+
+    private static void greet() {
+        System.out.println(ANSI_RED + "(Type \"bye\" to end the chat)" + ANSI_RESET);
+        say("Hello! This is Pat!");
+        say("What can I do for you?");
+    }
+
+    private static void bye() {
+        say("Bye! Remember, this is Pat!");
+    }
+
+    private static void addToTodo(Task task) {
+        todoList.add(task);
+        say("Added this task: ");
+        say(INDENT + task.getTask());
+        int size = todoList.size();
+        say("Now you have " + size + (size == 1 ? " task" : " tasks") + " in the list");
```

Comment:
I like how you notice the difference in singular vs plural forms of task. Perhaps can extract this (size == 1 ? " task" : " tasks") as a String variable